uidaho/squireproject
# Create and View Projects

Implement Projects:
- New model: `Project` in `projects` table. Fields: title, author, descriptions, body, create & modified
- Create Projects (`/create`) - "Start a New Project" nav element now works!
- View Projects (`/project/{id}`)
- Project Finder uses real projects and their attributes for display!
- Migration for project table: `php artisan migrate`
- Command for some test entries: `php artisan console:PopulateProjectsTable`

Still needed:
- Tests (?)
- Edit projects
- Delete projects
- All the other project stuff (i.e. social)

This will be ongoing, but wanted to get it in for demoing tomorrow. Fairly thoroughly debugged from my end, should be good.
